.github/workflows/test.yml

```diff
@@ -154,10 +145,9 @@ jobs:
           targets: wasm32-unknown-unknown
       - uses: taiki-e/install-action@cargo-hack
       - uses: Swatinem/rust-cache@v2
-      - run: |
-          cargo hack check --feature-powerset --optional-deps serde,rkyv \
-            --skip default --skip __internal_bench --skip __doctest \
-            --skip iana-time-zone --skip pure-rust-locales
+      - uses: actions/setup-node@v3
+      - uses: jetli/wasm-pack-action@v0.4.0
+      - run: TZ="$(date +%z)" NOW="$(date +%s)" wasm-pack test --node -- --features wasmbind
```

Comment:
Why do we need to set `TZ` and `NOW` here? Maybe add a comment?

Reply:
Added a comment